examples/example_vm/example_vm.cpp

```diff
@@ -5,33 +5,40 @@
 /// @file
 /// Example implementation of the EVMC VM interface.
 ///
-/// This VM does not do anything useful except for showing how EVMC VM API
-/// should be implemented. The implementation uses the C API directly
-/// and is done in simple C++ for readability.
+/// This VM implements a subset of EMV instructions in simplistic, incorrect and unsafe way:
+/// - memory bounds are not checked,
+/// - stack bounds are not checked,
+/// - most of the operations are done with 8-bit precision.
+/// Yet, it is capable of couping with some example EVM bytecode inputs, what is very useful
+/// in integration testing. The implementation is done in simple C++ for readability and uses
+/// pure C API and some C helpers.
 
 #include "example_vm.h"
+#include <evmc/evmc.h>
+#include <evmc/helpers.h>
 #include <cstdio>
 #include <cstdlib>
 #include <cstring>
 
-
+namespace example
+{
 /// The example VM instance struct extending the evmc_vm.
-struct example_vm : evmc_vm
+struct ExampleVM : evmc_vm
```

Comment:
I guess this renaming would be more part of the "modernize c++" commit.

Reply:
I'm thinking about dropping this commit as it is mostly replaced anyway.

Reply:
This commit is dropped now.